src/components/proofs/EmailProof.tsx

```diff
@@ -99,38 +119,51 @@ export default function () {
   return (
     <Line breakWords>
       <div className={proofLineContainer}>
-        <ToolTip position="bottom" text={popoverText} fitContainer>
+        <ToolTip
+          position="bottom"
+          text={popoverText}
+          fitContainer
+          disabled={!open}
+        >
           <div className={emailTitleContainer}>
             <div className={emailTitleLeft}>
-              <BodyText small>Work email</BodyText>
-              <QuestionMark small />
+              <ProofText>Work email</ProofText>
+              <div className={revealAnimation(open)}>
+                <QuestionMark small />
+              </div>
             </div>
             <button className={arrowContainer} onClick={() => setOpen(!open)}>
               {showButtonText && (
-                <span>{!domain ? 'Get started' : 'Set token'}</span>
+                <span className={getStartedText}>
+                  {!domain ? 'Get started' : 'Set token'}
+                </span>
               )}
-              <Arrow disabled vertical turnDown={open} />
+              <div className={width('w-4')}>
+                <Arrow pulseDisabled open={open} />
+              </div>
             </button>
           </div>
         </ToolTip>
         {open && (
           <>
-            <BadgeText>
-              {domain ? (
-                <>
-                  A token has been sent to ${email}. Copy the token and add it
-                  here to create zk proof. Or{' '}
-                  <button
-                    className={textDecoration('underline')}
-                    onClick={resetEmail}
-                  >
-                    re-enter email.
-                  </button>
-                </>
-              ) : (
-                `Add your work email and we’ll send you a token for that email. Then, use the token here to create zk proof.`
-              )}
-            </BadgeText>
+            <div className={margin('mt-4')}>
```

Comment:
After I changed the height of the container the spacing broke, so I added margin-top